src/catppuccin.scss

```
@@ -2,6 +2,62 @@
 @use "@catppuccin/palette/scss/catppuccin" as catppuccin;
 @use "@catppuccin/highlightjs/sass/_theme" as hljs;
 
+@mixin generate-clipboard-icon-filters($flavor) {
+  @if $flavor == "latte" {
+    --copy-button-filter: invert(71%)
+      sepia(8%)
+      saturate(435%)
+      hue-rotate(191deg)
+      brightness(90%)
+      contrast(88%);
+    --copy-button-filter-hover: invert(27%)
+      sepia(98%)
+      saturate(1766%)
+      hue-rotate(211deg)
+      brightness(96%)
+      contrast(100%);
+  } @else if $flavor == "frappe" {
+    --copy-button-filter: invert(51%)
+      sepia(22%)
+      saturate(392%)
+      hue-rotate(192deg)
+      brightness(88%)
+      contrast(89%);
+    --copy-button-filter-hover: invert(64%)
+      sepia(43%)
+      saturate(414%)
+      hue-rotate(185deg)
+      brightness(98%)
+      contrast(91%);
+  } @else if $flavor == "macchiato" {
+    --copy-button-filter: invert(50%)
+      sepia(14%)
+      saturate(616%)
+      hue-rotate(193deg)
+      brightness(87%)
+      contrast(87%);
+    --copy-button-filter-hover: invert(64%)
+      sepia(13%)
+      saturate(1528%)
+      hue-rotate(186deg)
+      brightness(105%)
+      contrast(91%);
+  } @else if $flavor == "mocha" {
+    --copy-button-filter: invert(44%)
+      sepia(10%)
+      saturate(814%)
+      hue-rotate(193deg)
+      brightness(96%)
+      contrast(81%);
+    --copy-button-filter-hover: invert(68%)
+      sepia(11%)
+      saturate(2148%)
+      hue-rotate(187deg)
+      brightness(103%)
+      contrast(96%);
+  }
+}
+
```

In order to make the filters a bit more maintainable in the long run, how do you feel about applying the same filter for frappe, macchiato and mocha? See below:

```suggestion
@mixin clipboard-icon($flavor) {
  @if $flavor == "latte" {
    --copy-button-filter: sepia(8%)
      saturate(435%)
      hue-rotate(191deg)
      brightness(90%)
      contrast(88%);
    --copy-button-filter-hover: invert(25%)
      sepia(98%)
      saturate(1766%)
      hue-rotate(211deg)
      brightness(96%)
      contrast(100%);
  } @else {
    --copy-button-filter: invert(44%)
      sepia(10%)
      saturate(814%)
      hue-rotate(193deg)
      brightness(96%)
      contrast(81%);
    --copy-button-filter-hover: invert(68%)
      sepia(11%)
      saturate(2148%)
      hue-rotate(187deg)
      brightness(103%)
      contrast(96%);
  }
}
```

I tested the above on the four flavours and I think it looks good.

---

Disregard this, @uncenter informed me that we already have generated filters over in `catppuccin/userstyles` to solve this problem. 

I've copied the values over and applied them to mdBook and they look good - [`a3c1968` (#154)](https://github.com/catppuccin/mdBook/pull/154/commits/a3c1968bb184e030ed5a92e2e028636d76c4e5ad)